src/Piral.Blazor.Tools/content/src/blazor.codegen

```diff
@@ -34,7 +34,10 @@ const sourceDir = resolve(wwwRootDir, '_framework');
 const projectAssets = require(resolve(objectsDir, pajson));
 
 const project = require(resolve(piralPiletFolder, pjson));
-const appdir = resolve(piralPiletFolder, 'node_modules', project.piral.name);
+let appdir = resolve(piralPiletFolder, 'node_modules', project.piral.name);
+if(!existsSync(appdir + '/app') && existsSync(appdir + '/dist')){
```

Comment:
Yeah this is a bit risky in general. It might be another directory (than `dist`), but I think we can leave it like that for now.